keras_nlp/src/utils/preset_utils.py

```diff
@@ -18,12 +18,18 @@
 import json
 import os
 import re
+import shutil
+import urllib
+
+import tensorflow as tf
```

Comment:
we are doing this conditional import of tensorflow right now like this

https://github.com/keras-team/keras-nlp/blob/5a734f9926af0d215246f4bc47738281e1006aba/keras_nlp/src/models/gpt2/gpt2_causal_lm_preprocessor.py#L15-L21

I'm going to switch it up in a future PR, but let's follow this pattern for now.